SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -544,28 +544,134 @@ void CFlowOutput::SetAnalyzeSurface(const CSolver* const*solver, const CGeometry
+
+      su2double MassFlow = config->GetSurface_MassFlow(iMarker_Analyze);
+      if (si_units)      cout << setw(20) << "Mf (kg/s): " << setw(15) << MassFlow;
+      else if (us_units) cout << setw(20) << "Mf (lbs/s): " << setw(15) << MassFlow;
+
+      su2double NormalVelocity = config->GetSurface_NormalVelocity(iMarker_Analyze);
+      if (si_units)      cout << setw(20) << "Vn (m/s): " << setw(15) << NormalVelocity;
+      else if (us_units) cout << setw(20) << "Vn (ft/s): " << setw(15) << NormalVelocity;
+
+      cout << endl;
+
+      su2double Uniformity = config->GetSurface_Uniformity(iMarker_Analyze);
+      if (si_units)      cout << setw(20) << "Uniformity (m/s): " << setw(15) << Uniformity;
+      else if (us_units) cout << setw(20) << "Uniformity (ft/s): " << setw(15) << Uniformity;
+
+      su2double SecondaryStrength = config->GetSurface_SecondaryStrength(iMarker_Analyze);
+      if (si_units)      cout << setw(20) << "Secondary (m/s): " << setw(15) << SecondaryStrength;
+      else if (us_units) cout << setw(20) << "Secondary (ft/s): " << setw(15) << SecondaryStrength;
+
+      cout << endl;
+
+      su2double MomentumDistortion = config->GetSurface_MomentumDistortion(iMarker_Analyze);
+      cout << setw(20) << "Mom. Distortion: " << setw(15) << MomentumDistortion;
+
+      su2double SecondOverUniform = config->GetSurface_SecondOverUniform(iMarker_Analyze);
+      cout << setw(20) << "Second/Uniform: " << setw(15) << SecondOverUniform;
+
+      cout << endl;
+
+      su2double Pressure = config->GetSurface_Pressure(iMarker_Analyze);
+      if (si_units)      cout << setw(20) << "P (Pa): " << setw(15) << Pressure;
+      else if (us_units) cout << setw(20) << "P (psf): " << setw(15) << Pressure;
+
+      su2double TotalPressure = config->GetSurface_TotalPressure(iMarker_Analyze);
+      if (si_units)      cout << setw(20) << "PT (Pa): " << setw(15) <<TotalPressure;
+      else if (us_units) cout << setw(20) << "PT (psf): " << setw(15) <<TotalPressure;
+
+      cout << endl;
+
+      su2double Mach = config->GetSurface_Mach(iMarker_Analyze);
+      cout << setw(20) << "Mach: " << setw(15) << Mach;
+
+      su2double Density = config->GetSurface_Density(iMarker_Analyze);
+      if (si_units)      cout << setw(20) << "Rho (kg/m^3): " << setw(15) << Density;
+      else if (us_units) cout << setw(20) << "Rho (lb/ft^3): " << setw(15) << Density*32.174;
+
+      cout << endl;
+
+      if (compressible || energy) {
+        su2double Temperature = config->GetSurface_Temperature(iMarker_Analyze);
+        if (si_units)      cout << setw(20) << "T (K): " << setw(15) << Temperature;
+        else if (us_units) cout << setw(20) << "T (R): " << setw(15) << Temperature;
+
+        su2double TotalTemperature = config->GetSurface_TotalTemperature(iMarker_Analyze);
+        if (si_units)      cout << setw(20) << "TT (K): " << setw(15) << TotalTemperature;
+        else if (us_units) cout << setw(20) << "TT (R): " << setw(15) << TotalTemperature;
```

Comment:
I'm used to seeing total quantities with "0" in textbooks.
```suggestion
        if (si_units)      cout << setw(20) << "T0 (K): " << setw(15) << TotalTemperature;
        else if (us_units) cout << setw(20) << "T0 (R): " << setw(15) << TotalTemperature;
```